packages/solid-crs-id-proxy/Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM node:14-alpine AS build
+LABEL org.opencontainers.image.source=https://github.com/cultuurnet/solid-poc
+ARG NPM_TOKEN
+ENV UPSTREAM_URL=https://nde.eu.auth0.com/
+ENV PROXY_URL=https://auth.netwerkdigitaalerfgoed.nl/
+WORKDIR /usr/src/id-proxy
+COPY package*.json ./
+COPY .npmrc ./
+RUN echo "//npm.pkg.github.com/:_authToken=$NPM_TOKEN" >> .npmrc && \
+    npm install && \
+    rm -f .npmrc
+COPY . .
+RUN npm run build
+EXPOSE 80
+ENTRYPOINT node ./scripts/generate-openid-configuration.js $UPSTREAM_URL $PROXY_URL /usr/src/id-proxy/openid-configuration.json $ENVIRONMENT && \ 
+node ./scripts/generate-keys.js /usr/src/id-proxy/jwks.json && \
+npm run start -- -u http://0.0.0.0 -U $UPSTREAM_URL -m . -c config/proxy-config.json -o ./openid-configuration.json -j ./jwks.json
```

Comment:
do we need `$ENVIRONMENT` here? What should it be set to?

Reply:
I'd set default value to `test` like `UPSTREAM_URL ` and `PROXY_URL ` on lines 4 & 5, but not sure what impact it has.

Reply:
No, you're correct. It's not necessary. Removing it.